packages/lib/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const BASE_URL = process.env.BASE_URL || `https://${process.env.VERCEL_URL}`;
```

Comment:
Extracted for reuse across projects

Reply:
It would be nice to move the rest of the constants here, I abstained to use them from lib in website because they were in web so far!

Reply:
Yes, we can do that in a follow up. Also the App Store branch already has this as well